packages/botonic-cli/templates/blank/webpack.config.js

```diff
@@ -51,6 +51,7 @@ const resolveConfig = {
   extensions: ['*', '.js', '.jsx', '.ts', '.tsx'],
   alias: {
     BotonicProject: path.resolve(__dirname, 'src'),
+    webchat: path.resolve(__dirname, 'src', 'webchat'),
```

Comment:
Why is this alias needed? When is it used?

Reply:
This is used in order to import `webchat` from an outer package, in our case `@botonic/react` with no need to use relative paths. Here the line in question: 
https://github.com/hubtype/botonic/blob/a76b97494d7deb0948ee05a6b00d9dc2390ca1d1/packages/botonic-react/src/webchat-entry.js#L3